test/example_test.py

```diff
@@ -83,11 +83,15 @@ def Test():
         "debug": ["net", "blk", "thin", "mempool", "req", "bench", "evict"],
         "blockprioritysize": 2000000  # we don't want any transactions rejected due to insufficient fees...
     }
-    # "--tmpdir=/ramdisk/test",
-    #t.main(["--nocleanup", "--noshutdown"], bitcoinConf, None)
-    eachConf = [bitcoinConf]*4
-    # eachConf[3]["maxlimitertxfee"] = None  # classic does not have this option
-    t.main(["--tmpdir=/ramdisk/test"], bitcoinConf, None)
+    # folder to store bitcoin runtime data and logs
+    tmpdir = "--tmpdir=/tmp/cashInterop"
+    
+    for arg in sys.argv[1:]:
+        if ("--tmpdir=" or "--tmpdir =") in arg:
```

Comment:
Does "--tmpdir =" ever match?  I think it won't because if you run
"./prog.py --tmpdir =foo"
then argv will be:
["prog.py", "--tmpdir", "=foo"]
(it splits by spaces) So it should never match

Reply:
It's removed. Thanks.